wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
Technically this would be optional, but you can actually pass an Enum directly in a DTO and then let ASP.NET deserialize it from the provided string.  If it is not a valid Enum value ASP.NET will provide a null value (you would therefore need to make sure this is a nullable Enum).  That would save the bit of code which parses and then sets it.  I view this as optional though since using a string does provide some additional flexibility in providing error messages since you would know what they provided, not just that it wasn't a valid enum.